src/cgame/rocket/rocketDataSelect.h

```diff
@@ -85,35 +86,36 @@ class RocketDataSelect : public Rml::ElementFormControlDataSelect, public Rml::E
 	{
 		extern std::queue< RocketEvent_t * > eventQueue;
 
-		if ( event.GetTargetElement() == owner && event == "show" )
```

Comment:
Is this guaranteed to not get events from children?

Reply:
Changed it back to if statements.